lib/host-sp-messages/src/lib.rs

```diff
@@ -336,11 +348,7 @@ mod tests {
             version: 123,
             sequence: 456,
         };
-        let host_to_sp = HostToSp::HostPanic {
-            status: 78,
-            cpu: 90,
-            thread: 12345,
-        };
+        let host_to_sp = HostToSp::HostPanic { code: 78 };
```

Comment:
Thanks for fixing these tests; I'll see if I can get them added to CI so they don't bitrot moving forward.